app.test.js

```diff
@@ -0,0 +1,10 @@
+import * as React from "react";
+import * as renderer from "react-test-renderer";
+import App from "./App";
+
+it("Renders the Footer correctly", () => {
+    const tree = renderer
+        .create(<App />)
```

Comment:
Did you have any success creating a snapshot of App? I had a really hard time trying to get the most basic thing going in Jest